apps/fyle/tasks.py

```
@@ -311,9 +315,82 @@ def update_non_exported_expenses(data: Dict) -> None:
             expense_state = 'NOT_EXPORTED'
 
         if expense_state and expense_state not in ['COMPLETE', 'IN_PROGRESS']:
+            # Store the is_skipped status
+            was_skipped = expense.is_skipped
```

Ruff (0.8.2)

Local variable `was_skipped` is assigned to but never used

Remove assignment to unused variable `was_skipped`

(F841)